jest.config.js

```diff
@@ -79,7 +79,115 @@ const testVenia = inPackage => ({
     // @magento namespaced packages like Peregrine and Venia UI as well, when
     // it's testing Venia. That way, changes in sibling packages don't require a
     // full compile.
-    transformIgnorePatterns: ['node_modules/(?!@magento/)']
+    transformIgnorePatterns: ['node_modules/(?!@magento/)'],
+    globals: {
+        TEST_GLOBAL: 'true',
+        UNION_AND_INTERFACE_TYPES: {
```

Comment:
This PR introduced a dependency on globals in a way that uncovered a bug in our config. The previous location of the `globals` configuration did not actually set globals and the test for `STORE_NAME` hardcoded it in the test spec rather than relying on the config (probably because it was broken).

This now properly creates serializable globals for Jest.